tests/baselines/reference/renameNumericalIndex.baseline.jsonc

```diff
@@ -0,0 +1,4 @@
+// === findRenameLocations ===
+// === /tests/cases/fourslash/renameNumericalIndex.ts ===
+// const foo = { <|[|0RENAME|]: true|> };
```

Comment:
I don't know if there's another method I can use to rename this into a valid identifier, since the rename results in `{ 0: true }` (valid code) -> `{ 0RENAME: true }` (invalid code)

Reply:
I don’t think `0RENAME` is real text here—I think it’s a baseline marker—but I find this baseline format very hard to read. In general, it appears possible to rename something to an invalid identifier in many ways, so I wouldn’t worry about invalid rename input like `0hello` in this PR.